### PR TITLE
Improved docs for better user and contributor experience (WIP)

### PR DESCRIPTION
**Pull request by Divy123**
Fixes #0000  (<=== Replace `0000` with the Issue Number)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts
* [ ] PR is descriptively titled
* [ ] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by Divy123** [comment]
@tech4GT just to confirm that was it dynamic module, we discussed on for improving its docs?

File: src/FormatInput.js [hunk 1]
```diff
@@ -1,3 +1,30 @@
+/*
+The file as the name suggests is used to format input as recieved from user
```
Comment (IgorWilbert) [inline src/FormatInput.js]: `received`

File: src/FormatInput.js [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+
+'+' : add a step
+'-' : remove a step
+'^' : insert a step (in between)
+'r' : run a step
+'l' : load image
+
+*/
+
+
```
Comment (IgorWilbert) [inline src/FormatInput.js]: I suggest removing some of these blank lines

**Comment by IgorWilbert** [comment]
Great work @Divy123 ! Left some comments, I hope they are helpful. Thanks!

File: src/FormatInput.js (hunk 1 above)
Comment (Divy123) [reply src/FormatInput.js]: Thanks :sweat_smile:

File: src/FormatInput.js (hunk 2 above)
Comment (Divy123) [reply src/FormatInput.js]: Yeah sure!
Thanks!!

**Comment by Divy123** [comment]
@jywarren can you please help me with dynamic module. Can you please explain its working a bit so that I can work more on its docs.

**Comment by HarshKhandeparkar** [comment]
Outdated. Closing for now, feel free to rebase and reopen :)

**Pull request closed without merging**